user-interface/src/case-detail/panels/CaseDetailOverview.tsx

```diff
@@ -290,97 +290,91 @@ export default function CaseDetailBasicInfo(props: CaseDetailBasicInfoProps) {
-                          className="usa-link case-detail-item-value"
-                          data-testid={`case-detail-consolidation-link`}
-                        />{' '}
-                        <span className="case-detail-title-value">
-                          {caseDetail.consolidation[0].otherCase.caseTitle}
-                        </span>
-                      </>
-                    )}
-                  </div>
+              <div className="consolidation case-card">
+                <h4>{consolidationTypeMap.get(caseDetail.consolidation[0].consolidationType)}</h4>
+                <div>
                   {caseDetail.consolidation[0].documentType === 'CONSOLIDATION_FROM' && (
-                    <div>
-                      <span className="case-detail-consolidated-case-count">
-                        Cases Consolidated: {caseDetail.consolidation.length + 1}
+                    <span className="case-detail-item-name">Lead Case: (this case)</span>
+                  )}
+                  {caseDetail.consolidation[0].documentType === 'CONSOLIDATION_TO' && (
+                    <>
+                      <span className="case-detail-item-name">Lead Case:</span>
+                      <CaseNumber
+                        caseId={caseDetail.consolidation[0].otherCase.caseId}
+                        className="usa-link case-detail-item-value"
+                        data-testid={`case-detail-consolidation-link`}
+                      />{' '}
+                      <span className="case-detail-title-value">
+                        {caseDetail.consolidation[0].otherCase.caseTitle}
                       </span>
-                    </div>
+                    </>
                   )}
+                </div>
+                {caseDetail.consolidation[0].documentType === 'CONSOLIDATION_FROM' && (
                   <div>
-                    <span className="case-detail-item-name">Order Filed:</span>
-                    <span
-                      className="case-detail-item-value"
-                      data-testid={`case-detail-consolidation-order`}
-                    >
-                      {/* This order date is not likely the correct one.  Clarification from Phoenix has been requested */}
-                      {formatDate(caseDetail.consolidation[0].orderDate)}
+                    <span className="case-detail-consolidated-case-count">
+                      Cases Consolidated: {caseDetail.consolidation.length + 1}
                     </span>
                   </div>
+                )}
+                <div>
+                  <span className="case-detail-item-name">Order Filed:</span>
+                  <span
+                    className="case-detail-item-value"
+                    data-testid={`case-detail-consolidation-order`}
+                  >
+                    {/* This order date is not likely the correct one.  Clarification from Phoenix has been requested */}
```

Comment:
Is this something to address now or in a later PR?

Reply:
Though it looks like according to Git, that I added that comment, I do not believe I did.  I'm not sure what that is about.

Reply:
Looks like it was added to `CaseDetailBasicInfo.tsx` back in March. I would think we could remove the comment unless we want to dig into it.